src/ECER.Resources.Documents/Applications/ApplicationChildren/ProfessionalDevelopments.cs

```diff
@@ -149,6 +149,26 @@ private async Task AddFilesForProfessionalDevelopment(ecer_ProfessionalDevelopme
     }
   }
 
+  public async Task<string> AddProfessionalDevelopmentForSubmittedApplication(ProfessionalDevelopment newProfessionalDevelopment, string applicationId, string userId, CancellationToken cancellationToken)
```

Comment:
Why did we not use the existing functionality? UpdateProfessionalDevelopments function?

Reply:
1) UpdateProfessionalDevelopments function deletes the existing professional developments and then adds new ones on top, for submitted applications its a simple addition wherein we just need to add additional professional developments without touching the existing ones.
2) it provides for better separation of concerns (in progress applications vs submitted applications) just like UpdateWorkExReferenceForSubmittedApplication function

Reply:
The UpdateProfessionalDevelopments function only deletes entries removed from updatedEntities; otherwise, existing ones remain untouched.

A better approach would be to retrieve existing professional developments in the manager layer, add new ones, and pass the combined list to the function. This adds new entries without modifying existing ones.

However, this duplicates functionality and doesn't improve separation of concerns. If we’ve done this in UpdateWorkExReferenceForSubmittedApplication, we should refactor for consistency.

Reply:
Refactored Add professional developments for submitted application, using handler to add new professional development to existing list of professional development and using existing SaveApplication repository function